docs/ADVANCED.md

```diff
@@ -209,17 +209,35 @@ Here's an overview of what this looks like if you want to persist *all* your ind
 ---
 splunk:
   smartstore:
-    - indexName: default
-      remoteName: remote_store
-      scheme: s3
-      remoteLocation: <bucket-name>
-      s3:
-        access_key: <access_key>
-        secret_key: <secret_key>
-        endpoint: http://s3-us-west-2.amazonaws.com
+    index:
+      - indexName: default
+        remoteName: remote_store
+        scheme: s3
+        remoteLocation: <bucket-name>
+        s3:
+          access_key: <access_key>
+          secret_key: <secret_key>
+          endpoint: http://s3-us-west-2.amazonaws.com
   ...
 ```
 
+Some cache management options are also available. Options defined under the index stanza correspond to options in `index.conf` https://docs.splunk.com/Documentation/Splunk/8.0.0/admin/Indexesconf. While options defined outside the index correspond to options in `server.conf` https://docs.splunk.com/Documentation/Splunk/8.0.0/admin/Serverconf, note that currently only `[cachemanager]` stanza is supported. This is an example config that defines cache settings and retention policy:
```

Comment:
index.conf -> indexes.conf here

Reply:
Also, maybe change the URL for the docs link so that it references `latest` instead of `8.0.0`?

Reply:
oohlala, changin it